### ISSUE TITLE
Forge?

### ISSUE THREAD

**Issue by itsdefter** [FXCourel/Copper-Rails#7]
Any plan for adding forge and neoforge support for different versions (like forge 1.18.2 and neoforge 1.20.1)?

**Comment by FXCourel** [comment]
No. I don't have time to learn forge and I don't use it, as fabric is much more used in newest versions (though I don't know if neoforge achieved to make people in forge more interested). An other reason is that the mod was a somewhat a "port" of the forge mod Modern Minecarts.
